src/e3/anod/checkout.py

```diff
@@ -148,6 +148,10 @@ def update_external(
             if p.status != 0:
                 raise e3.error.E3Error("rsync failed")
         else:
+            # Test for a git repository by looking for ".git" in the current
+            # directory - either as a file or as a directory. We check both
+            # because, if a normal git clone, ".git" is a directory; if a
+            # git clone with `git submodule init``, ".git" is a file.
```

Comment:
Small typo (two backquotes after the git command).

Reply:
Over-eager helpfulness by VS Code, I suspect ("here, I'll add a closing quote, too!"). Thanks - I fixed it. :)